src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/18.diff.sql

```diff
@@ -0,0 +1,299 @@
+﻿SET XACT_ABORT ON;
+
+BEGIN TRANSACTION
```

Comment:
As the best practice, we don't wrap migration script(diff script) in a transaction as few statements can hold exclusive lock on the table and if we have expensive index creation statements that follows, this can result in blocking for extended periods of time. That is the reason we make migration script idempotent instead of transactional.

So can we remove top level transaction and make each script idempotent if it is not and include transaction within the individual script if required?